hal/lib/athena/ChipObject.h

```diff
@@ -13,12 +13,6 @@
 #include <stdint.h>
 
 #include "FRC_FPGA_ChipObject/RoboRIO_FRC_ChipObject_Aliases.h"
```

Comment:
I'm pretty sure it originally came from NI, but the point remains regardless. We can't mess with the order in this file.